service.go

```diff
@@ -216,8 +216,16 @@ func (s *service) eventHandler() {
 			if strings.HasSuffix(ev.EventType(), "start") {
 				s.startRecording(channelID, teamID)
 			} else {
-				go s.stopRecording(channelID, teamID)
+				go s.stopRecording(channelID)
```

Comment:
If `teamID` isn't needed, we can remove the cast and check in lines 210-214?

Reply:
The way the code is written we need `teamID` for starting a recording, not for stopping it. We could make an extra API call to fetch the team from the channel but was trying to avoid that.

Reply:
Ah! I completely missed that we need it to start, apologies.